misc/xml-to-record-converter/src/main/java/io/ballerina/xmltorecordconverter/util/ConverterUtils.java

```diff
@@ -71,6 +72,45 @@ public static String escapeIdentifier(String identifier) {
         }
     }
 
+    /**
+     * This method returns the SyntaxToken corresponding to the JsonPrimitive.
+     *
+     * @param value JsonPrimitive that has to be classified.
+     * @return {@link Token} Classified Syntax Token.
+     */
+    public static Token getPrimitiveTypeName(String value) {
+        if (isBoolean(value)) {
+            return AbstractNodeFactory.createToken(SyntaxKind.BOOLEAN_KEYWORD);
+        } else if (isInteger(value)) {
+            return AbstractNodeFactory.createToken(SyntaxKind.INT_KEYWORD);
+        } else if (isDouble(value)) {
+            return AbstractNodeFactory.createToken(SyntaxKind.DECIMAL_KEYWORD);
+        }
+        return AbstractNodeFactory.createToken(SyntaxKind.STRING_KEYWORD);
```

Comment:
Can we safely assume that the `else` case will only be strings?

Reply:
Yes right, actually we can  consider anything in XML as String(Anything in what we are processing), so it's safe to conclude as Strings otherwise.